### DISCUSSION TITLE
Display of egsphant file

### DISCUSSION THREAD

**Discussion by subha18** [nrc-cnrc/EGSnrc#997] [General]
I am having trouble in viewing of the egsphant file, created from the DICOM images using CTCREATE. I tried with egs_view and VICTORIA both they look distorted. Can not make out anything out of it. However the dose calculations using the egsphant file is fine.

How can i generate similar egsphant file along with the contours like the one PeppaBreastHDR192Ir_MBDCA-WG.egsphant supplied with the distribution of egs_brachy code.

**Comment by mchamberland** [comment]
Can you post a screenshot and perhaps the file, or at least the beginning of the file (the part with the media and voxel definitions at least).

**Comment by subha18** [comment]
Thanks you very much for your quick response.

Attached are the egsphant file and the display images
[Test_dicom.egsphant.gz](https://github.com/nrc-cnrc/EGSnrc/files/11574873/Test_dicom.egsphant.gz)
![image_egs_view](https://github.com/nrc-cnrc/EGSnrc/assets/35722513/b264381e-606a-443e-96e0-3a341eac7ddb)
![Image_VICTORIA](https://github.com/nrc-cnrc/EGSnrc/assets/35722513/6bf85756-3913-4cc4-820a-641e63c3fa80)

**Reply by mchamberland** [reply]
I have seen that before. I can't remember what causes it now...

Are you still using egs_brachy? If so, could you try creating your egsphant using eb_gui instead of ctcreate? It can be downloaded from GitHub [here](https://github.com/clrp-code/eb_gui).

**Reply by subha18** [reply]
Hi Marc,
I have truncated few slices to reduce the file size below so that i can upload it here. Yes, dose calculations are done in egs_brachy which is fine only viewing the phantom is an issue.

**Comment by rtownson** [comment]
What's the resolution of the DICOM CT data? If it is not 512x512, you may need to adjust MAX_CT_COL_SIZE in `HEN_HOUSE/omega/ctcreate/ReadCTDICOM.c` and `$CTJMAX` in `ctcreate.mortran`.

**Reply by subha18** [reply]
Thanks for the reply. 
Yes it is 512 x 512 with few slices. Even if i created the egsphant file using all the slices  it looks similar in egs_view as well as in VICTORIA.

I was wondering how can i generate similar egsphant file containing the contours like the one PeppaBreastHDR192Ir_MBDCA-WG.egsphant supplied with the distribution of egs_brachy code. I believe it is not possible using ctcreate.

**Reply by rtownson** [reply]
Try using eb_gui.

Can you post the output text from ctcreate here?

**Reply by subha18** [reply]
[interactiv.ctlst.txt](https://github.com/nrc-cnrc/EGSnrc/files/11598790/interactiv.ctlst.txt)
I have not yet tried with eb_gui. Will try 

Thanks
